crates/stackable-versioned-macros/fixtures/inputs/default/submodule.rs

```diff
@@ -0,0 +1,11 @@
+#[versioned(version(name = "v1alpha1"), version(name = "v1"))]
+// ---
+mod versioned {
+    mod v1alpha1 {
+        pub use my::reexport::v1alpha1::*;
+    }
+
+    struct Foo {
+        bar: usize,
+    }
+}
```

Comment:
Just having some fun, inventing our own syntax:

```suggestion
#[versioned(version(name = "v1alpha1"), version(name = "v1"))]
// ---
mod versioned {
    imports for v1alpha1 {
        my::reexport::v1alpha1::*;
    }

    struct Foo {
        bar: usize,
    }
}
```

_You can completely ignore this_